.tekton/basic-component-pipeline.yaml

```diff
@@ -356,6 +356,30 @@ spec:
       operator: in
       values: [ "false" ]
 
+  - name: roxctl-image-scan
+    params:
+    - name: image
+      value: $(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)
+    - name: output_format
+      value: table
+    - name: rox_image
+      value: quay.io/stackrox-io/roxctl:4.4.2
+    runAfter:
+    - build-container
+    taskRef:
+      params:
+      - name: name
+        value: rhacs-image-scan
+      - name: bundle
+        value: gcr.io/tekton-releases/catalog/upstream/rhacs-image-scan@sha256:6be52f9e623e3fecbf2d2b542ac59cc9c6993311306e78651aba98f4e520aa6c
```

Comment:
1. Since this is not part of <https://github.com/konflux-ci/build-definitions>, could you kindly link a place where it is documented? I'm used to find things in that repo `konflux-ci/build-definitions`, something new may confuse me and maybe others :-)
2. There are already `acs-*` tasks in <https://github.com/konflux-ci/build-definitions/tree/main/task> why aren't those used and what's going to be their faith? I recall @rhybrillou asked about them in Slack and not sure what happened after and are we taking ownership of these.

Reply:
1. It's documented in the tekton catalog https://github.com/tektoncd/catalog/tree/main/task/rhacs-image-scan.
2. We are currently in the unfortunate situation that we maintain our Tekton tasks in the official catalog repo (https://github.com/tektoncd/catalog/tree/main/task/rhacs-image-scan). However, the Konflux team created another set up tasks in their "trusted" repository https://github.com/konflux-ci/build-definitions/tree/main/task. These tasks have now diverged since new features have been added. The reason I went with "our" tasks here is that we want to dog food our tasks including their new features.

In my opinion we should strive to reconcile these tasks to avoid duplication. I talked to the Konflux team about this and created https://issues.redhat.com/browse/KONFLUX-3156. So my proposal would be to (eventually) deprecate and drop the `konflux-ci/build-definitions` tasks.

Reply:
Pursuing your proposal, could you please send a PR towards https://github.com/konflux-ci/build-definitions/ add a warning note that Tekton catalog tasks should be used instead of Konflux ones?